### PR TITLE
crimson/mgr: Fix config show command

### PR DESCRIPTION
**Pull request by amathuria**
Update the MMgrOpen message to include config_bl and config_defaults_bl

Fixes: https://tracker.ceph.com/issues/61846

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by ronen-fr** [comment]
https://tracker.ceph.com/issues/61846 is empty of any description?

File: src/crimson/mgr/client.h [hunk 1]
```diff
@@ -35,6 +35,7 @@ class Client : public crimson::net::Dispatcher {
   seastar::future<> start();
   seastar::future<> stop();
   void report();
+  uint64_t last_config_bl_version = 0;
```
Comment (Matan-B) [inline src/crimson/mgr/client.h:38]: Should be private

Comment (Matan-B) [inline src/crimson/mgr/client.h:38]: Note: this will also be used once `MMgrReport` is supported.

**Review by Matan-B — APPROVED** [review]
LGTM. few nits.

**Comment by Matan-B** [comment]
> https://tracker.ceph.com/issues/61846 is empty of any description?

IIUC, the title should do. The tracker is opened for backport maintainability.

File: src/crimson/common/config_proxy.h [hunk 2]
```diff
@@ -96,6 +96,14 @@ class ConfigProxy : public seastar::peering_sharded_service<ConfigProxy>
     return values.get();
   }
 
+  void get_config_bl(uint64_t have_version,
```
[the remark this answers is not in the document]
Comment (Matan-B) [reply src/crimson/common/config_proxy.h:99]: To comply with `md_config_t::get_config_bl`

**Pull request closed without merging**